dockerfiles/Dockerfile

```diff
@@ -6,6 +6,15 @@ RUN apk add --no-cache \
 
 FROM scratch
 LABEL "com.centurylinklabs.watchtower"="true"
+LABEL "org.opencontainers.image.url"="https://containrrr.dev/watchtower/" \
+      "org.opencontainers.image.documentation"="https://containrrr.dev/watchtower/" \
+      "org.opencontainers.image.source"="https://github.com/containrrr/watchtower" \
+      "org.opencontainers.image.licenses"="Apache-2.0" \
+# unsure about capitalization
+      "org.opencontainers.image.title"="watchtower" \
```

Comment:
Are there any feelings about the capitalization of watchtower? As a non-native speaker I am a bit unsure.